convert_osm/src/extract.rs

```diff
@@ -539,7 +540,7 @@ fn is_bldg(tags: &Tags) -> bool {
 
 fn get_bldg_amenities(tags: &Tags) -> Vec<Amenity> {
     let mut amenities = Vec::new();
-    for key in vec!["amenity", "shop", "craft", "office", "tourism", "leisure"] {
+    for key in ["amenity", "shop", "craft", "office", "tourism", "leisure"] {
```

Comment:
Hmm, with 1.52.1, I'm getting "arrays are not iterators, but slices like the following are: `&[1, 2, 3]`". I thought I saw this in the recent release notes, but maybe not. Are you using nightly?

Reply:
Yes, this might be the new array iterator impl on nightly. I'll change it to the backwards-compatible `for &key in &[...]`.

Reply:
Fixed.